yarn-project/aztec.js/src/account_manager/index.ts

```diff
@@ -128,17 +134,24 @@ export class AccountManager {
       }
       await this.#register();
       const encryptionPublicKey = this.getEncryptionPublicKey();
+      const { chainId, protocolVersion } = await this.pxe.getNodeInfo();
+      const deployWallet = new SignerlessWallet(this.pxe, new DefaultMultiCallEntrypoint(chainId, protocolVersion));
+
+      await this.#register();
```

Comment:
Do we need to call this again? It's 5 lines above.

Reply:
No, I missed this duplication after running `gt squash`